pkg/render/fluentd.go

```diff
@@ -182,17 +180,18 @@ func (c *fluentdComponent) ResolveImages(is *operatorv1.ImageSet) error {
 	path := c.cfg.Installation.ImagePath
 	prefix := c.cfg.Installation.ImagePrefix
 
-	if c.cfg.OSType == rmeta.OSTypeWindows && c.cfg.EnterpriseVariant {
+	if c.cfg.OSType == rmeta.OSTypeWindows && c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
 		var err error
-		c.image, err = components.GetReference(components.ComponentFluentdWindows, reg, path, prefix, is)
+		c.image, err = components.GetReference(components.ComponentTigeraFluentdWindows, reg, path, prefix, is)
 		return err
 	}
 
 	var err error
-	if c.cfg.EnterpriseVariant {
-		c.image, err = components.GetReference(components.ComponentFluentd, reg, path, prefix, is)
+	if c.cfg.Installation.Variant == operatorv1.TigeraSecureEnterprise {
+		c.image, err = components.GetReference(components.ComponentTigeraFluentd, reg, path, prefix, is)
 	} else {
-		c.image, err = components.GetReference(components.ComponentCalicoFluentd, reg, path, prefix, is)
+		//c.image, err = components.GetReference(components.ComponentCalicoFluentd, reg, path, prefix, is)
```

Comment:
Please revert this change so it doesn't get forgotten.

If you want to make changes to the images that are used, you can modify `config/calico_versions.yaml` and run `make gen-versions` instead.